src/App.js

```diff
@@ -1,13 +1,109 @@
 import React from 'react';
-
+import uuid from 'uuid';
+import './App.css'
+import Todo from './components/TodoComponents/Todo'
+import TodoForm from './components/TodoComponents/TodoForm'
+import SearchTodo from './components/TodoComponents/searchTodo'
+const todoData = [
+  {
+    task: 'Organize Garage',
+    id: 1528817077286,
+    completed: false
+  },
+  {
+    task: 'Bake Cookies',
+    id: 1528817084358,
+    completed: false
+  }
+];
+const getTodoData= window.localStorage.getItem('todoData');
+console.log(getTodoData)
+let datas = []
+if (getTodoData === null || getTodoData === [] || getTodoData === undefined){
+   window.localStorage.setItem('todoData', JSON.stringify(todoData)) 
+   const tododatas= window.localStorage.getItem('todoData')
```

Comment:
All this section here is working flawlessly, it feels a bit chaotic and not really polished though. There are few fixes that can be made here between line 22 and 28.